x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/IndexFollowingIT.java

```diff
@@ -101,9 +102,24 @@ public void testFollowIndex() throws Exception {
         assertAcked(leaderClient().admin().indices().prepareCreate("index1").setSource(leaderIndexSettings, XContentType.JSON));
         ensureLeaderYellow("index1");
 
-        final int firstBatchNumDocs = randomIntBetween(2, 64);
+        final int firstBatchNumDocs = randomIntBetween(800, 1200);
```

Comment:
The total test takes 1-2 seconds to run with 800-1200 documents. But `rarely` seems fine since we just want this to fail every once in a while if there was a problem.

I changed this to:

```
        // Sometimes we want to index a lot of documents to ensure that the recovery works with larger files
        if (rarely()) {
            firstBatchNumDocs = randomIntBetween(1800, 2000);
        } else {
            firstBatchNumDocs = randomIntBetween(10, 64);
        }
```